src/Disqord.Rest/ActionProperties/Create/Channel/CreatePrivateThreadChannelActionProperties.cs

```diff
@@ -0,0 +1,9 @@
+using Qommon;
+
+namespace Disqord
+{
+    public sealed class CreatePrivateThreadChannelActionProperties : CreateThreadChannelActionProperties
+    {
+        public Optional<bool> Invitable { internal get; set; }
```

Comment:
```suggestion
        public Optional<bool> AllowsInvitation { internal get; set; }
```
Be consistent with the [existing property names](https://github.com/Quahu/Disqord/blob/bot-rewrite/src/Disqord.Core/Entities/Core/Channels/Guild/Threads/IThreadMetadata.cs#L37).